lib/api/controllers/securityController.js

```diff
@@ -113,7 +112,7 @@ const persistUser = Bluebird.coroutine(function* persistUserGenerator (kuzzle, r
     }
   };
   // Throw in case of failure with the right KuzzleError object
-  const createdUser = yield kuzzle.repositories.user.hydrate(new User(), pojoUser)
+  const createdUser = yield kuzzle.repositories.user.fromDTO(pojoUser)
```

Comment:
DTO?

Reply:
Data Transfer Object, I guess ?

Reply:
If so, wouldn't it be clearer to rename `fromDTO`/`toDTO` to either `unserialize`/`serialize` or `unmarshal`/`marshal`?

Reply:
Not to me. serialization/marshalling/pickling is to me the process of transforming some data into a representation of this data, usually either a string or some binary format.
DTO are objects meant to transport data, which is exactly what they achieve here.